www/class/config-generate-remote/Host.php

```diff
@@ -49,6 +49,8 @@ private function getHostGroups(array &$host)
                 $this->stmtHg = $this->backendInstance->db->prepare("SELECT
                     hostgroup_hg_id
                 FROM hostgroup_relation
+                INNER JOIN hostgroup ON hg_id = hostgroup_hg_id
```

Comment:
this has nothing to do with the missing require and the typo correction... 
please, could you add it to the description in the template ?